### PR TITLE
Add Issue and Pull Request Templates

### PR DESCRIPTION
**Pull request by jonstacks**
## What

Add Issue and Pull Request Templates

## Why

Will help us better triage issues by providing community members and ourselves templates for creating issues. We can still fall back to "Blank" issues if we need to.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/helm_bug_report.yaml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+      Please provide the context in which the problem occurred and explain what happened
+  validations:
+    required: true
+- type: textarea
+  attributes:
+    label: What you think should happen instead
+    description: What do you think went wrong?
+    placeholder: >
+      Please explain why you think the behavior you observed is incorrect. It is extremely helpful
+      if you copy&paste the fragment of logs showing the exact error messages or wrong behavior.
```
Comment (deroine) [inline .github/ISSUE_TEMPLATE/helm_bug_report.yaml:46]:
```suggestion
      if you copy & paste the fragment of logs showing the exact error messages or wrong behavior.
```

File: .github/ISSUE_TEMPLATE/feature_request.yaml [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+- type: textarea
+  attributes:
+    label: Description
+    description: A short description of the feature request
+- type: textarea
+  attributes:
+    label: Use Case
+    description: What is the use case for this feature?
+    placeholder: >
+      Rather than telling us how you might implement this feature, try to take a step
```
Comment (deroine) [inline .github/ISSUE_TEMPLATE/feature_request.yaml:22]: love this

**Review by deroine — APPROVED** [review]
Looks good to me

File: .github/ISSUE_TEMPLATE/helm_bug_report.yaml [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+  validations:
+    required: true
+- type: textarea
+  attributes:
+    label: Helm Chart configuration
+    description: Additional description of your Helm Chart configuration.
+    placeholder: >
+      Enter any relevant details of your Helm Chart configuration. Maybe you can
+      paste your `values.yaml` or important parts of it here? Make sure to surround the code
+      you paste with ``` ```.
```
Comment (alex-bezek) [inline .github/ISSUE_TEMPLATE/helm_bug_report.yaml:31]: some of these (if not all) seem like they would also be useful for the controller bug report. I wonder how accurate these will be reported fwiw as many problems would probably be some combination of the 2?

File: .github/ISSUE_TEMPLATE/helm_bug_report.yaml [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+- type: textarea
+  attributes:
+    label: How to reproduce
+    description: What should we do to reproduce the problem?
+    placeholder: >
+      Please provide a reproducible step-by-step case of how to reproduce the problem as minimally and
+      precisely as possible. It will greatly help us in figuring out the root cause of the problem and fixing it.
+- type: markdown
+  attributes:
+    value: "Thank you for help in improving the Ngrok Ingress Controller Helm Chart!"
```
Comment (alex-bezek) [inline .github/ISSUE_TEMPLATE/helm_bug_report.yaml:56]: nit: `Ngrok` is always lower case is my understanding. You can see on the main site its always just `ngrok`

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+are covered with tests. In case of new features or big changes remember
+to adjust the documentation.
+
+In case of an existing issue, reference it using one of the following:
+
+closes: #ISSUE
+related: #ISSUE
+
+How to write a good git commit message:
+http://chris.beams.io/posts/git-commit/
```
Comment (alex-bezek) [inline .github/PULL_REQUEST_TEMPLATE.md:11]: saw the `http` and was just curious
```
curl -I http://chris.beams.io/posts/git-commit/
HTTP/1.1 301 Moved Permanently
Server: nginx
Date: Tue, 13 Dec 2022 16:40:06 GMT
Content-Type: text/html
Content-Length: 162
Connection: keep-alive
Location: https://cbea.ms/posts/git-commit/
```

**Review by alex-bezek — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/helm_bug_report.yaml (hunk 4 above)
Comment (deroine) [reply .github/ISSUE_TEMPLATE/helm_bug_report.yaml:56]: Good catch, I'd say not a nit, and we will want to change that.

File: .github/ISSUE_TEMPLATE/controller_bug_report.yaml [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+      Please provide the context in which the problem occurred and explain what happened
+  validations:
+    required: true
+- type: textarea
+  attributes:
+    label: What you think should happen instead
+    description: What do you think went wrong?
+    placeholder: >
+      Please explain why you think the behavior you observed is incorrect. It is extremely helpful
+      if you copy&paste the fragment of logs showing the exact error messages or wrong behavior.
```
Comment (deroine) [inline .github/ISSUE_TEMPLATE/controller_bug_report.yaml:26]:
```suggestion
      if you copy & paste the fragment of logs showing the exact error messages or wrong behavior.
```

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 5 above)
Comment (jonstacks) [reply .github/PULL_REQUEST_TEMPLATE.md:11]: I was drawing some inspiration from https://raw.githubusercontent.com/apache/airflow/main/.github/PULL_REQUEST_TEMPLATE.md

File: .github/ISSUE_TEMPLATE/helm_bug_report.yaml (hunk 4 above)
Comment (jonstacks) [reply .github/ISSUE_TEMPLATE/helm_bug_report.yaml]: @deroine, fixed in https://github.com/ngrok/ngrok-ingress-controller/pull/72/commits/8fc64dfce1b49f192ed48dc4ffec983bfe5d6933

File: .github/ISSUE_TEMPLATE/helm_bug_report.yaml (hunk 3 above)
Comment (jonstacks) [reply .github/ISSUE_TEMPLATE/helm_bug_report.yaml:31]: Yeah. I'm hoping some things like annotations on ingress objects will be clearly in the controller realm and things like the helm install not working going into the helm bug report. For things that are a combination of the two, I think we will definitely need to triage them appropriately.

**Pull request merged**